lab-jessica/lib/client.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+const uuid = require('node-uuid');
+
+const Client = function(socket){
```

Comment:
Probably best to just put module.exports here as the target of your function assignment. I know it was starter code, but you're welcome to improve it! ;-)